tests/construction_test.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+
+from honeybee_energy.material.opaque import EnergyMaterial
+from honeybee_energy.material.glazing import EnergyWindowMaterialGlazing
+from honeybee_energy.material.gas import EnergyWindowMaterialGas
+
+from honeybee_energy.construction import OpaqueConstruction, WindowConstruction
+
+import unittest
+import pytest
+
+
+class MaterialTestCase(unittest.TestCase):
```

Comment:
Any reason you are going back to using unittest? Not that I have any issues with the approach but is this something that you want to follow for other repos or just a place holder for now.

Reply:
This is what I have done on all of my repos.  If this one is backward, can you point me to an example of "the way forward"?

Reply:
Just using methods as suggested for pytest? Here is an example:

https://github.com/ladybug-tools/honeybee-radiance/blob/development/tests/plastic_material_test.py

Reply:
I can write the tests that way but I still don't see how they are run. Is there some other file that interacts with this one to call each of the test functions?

Reply:
You can run all tests running `python -m pytest .\tests` on Windows. This is how we already run all the tests on travis:

https://github.com/ladybug-tools/honeybee-energy/blob/29688c62d8655ae73bb7a408dfa163ae68c6d0bd/.travis.yml#L11

See pytest test discovery conventions here:

https://docs.pytest.org/en/latest/goodpractices.html#conventions-for-python-test-discovery